experimental/dds/tree2/src/core/undo/undoRedoManager.ts

```diff
@@ -192,29 +192,34 @@ export class UndoRedoManager<TChange, TEditor extends ChangeFamilyEditor> {
 	/**
 	 * Updates the state of this {@link UndoRedoManager} to correctly reference commits that have been rebased after merging.
 	 * @param newCommits - all commits which were appended to the source branch.
+	 * @param change - the change to the forest caused by the rebasing.
 	 * @param mergedUndoRedoManager - the {@link UndoRedoManager} of the branch that was merged.
 	 */
 	public updateAfterMerge(
 		newCommits: GraphCommit<TChange>[],
+		change: TChange | undefined,
```

Comment:
Consider replacing this with a boolean `forestChanged` or `changed` or something. These functions only care whether or not this argument is undefined, they don't care about what it is when it is defined. Even if they wanted to they couldn't, because TChange is generic and URM knows nothing about what it actually is.